Nativing/accounts/models.py

```diff
@@ -1,3 +1,87 @@
 from django.db import models
+from django.contrib.auth.models import PermissionsMixin, AbstractBaseUser, BaseUserManager
 
-# Create your models here.
+
+class CustomAccountManager(BaseUserManager):
+    def create_user(
+        self,
+        name, 
+        email, 
+        nickname,
+        date_of_birth,
+        user_image,
+        user_gender,
+        password=None
+    ):
+        user = self.model(
+            name=name,
+            email=email, 
+            nickname=nickname, 
+            date_of_birth=date_of_birth,
+            user_image=user_image,
+            user_gender=user_gender,
+        )
+        user.set_password(password)
+        user.is_superuser = False
+        user.save(using=self._db)
+        return user
+
+    def create_superuser(
+        self,
+        name, 
+        email, 
+        nickname, 
+        password
+    ):
+        user = self.model(
+            email=email, 
+            name=name,
+            nickname=nickname, 
+            password=password)
```

Comment:
```suggestion
            password=password
            # 1번 (null = true를 안할 경우)
            date_of_birth = 970117,
            user_image = None,
            user_gender = 'male'
            
            # 2번 (null = true를 한 경우)
            date_of_birth = None,
            user_image = None,
            user_gender = None,
            )
```
아까 위에서 언급했듯이, 커스텀 user 생성 메소드를 받아와서 superuser를 만드는 거라서 
몇가지 None값을 주던, 임의의 값을 주던 해줘야 정상적으로 생성됩니다.